### PR TITLE
Add link to change log to ACP update message

### PR DESCRIPTION
**Pull request by Kamahl19**
We should add a link to changelog to ACP when text about new version is displayed. It could help persuade admins to update

http://phpbbsocialnetwork.com/viewtopic.php?f=6&t=4799

### CODE REVIEW

File: root/adm/style/socialnet_update.html [hunk 1]
```diff
@@ -2,10 +2,10 @@
 	<p>{L_SN_VERSION_CHECK_EXPLAIN}</p>
 
 	<!-- IF B_COULD_NOT_GET_UPDATE -->
-		<div class="errorbox">
+  	<div class="errorbox">
```
Comment (senky) [inline root/adm/style/socialnet_update.html]: If you want to replace it, the only allowed indention character is tab, not spaces.

**Comment by Culprit-CZ** [comment]
MA

**Comment by senky** [comment]
MA

**Pull request merged**